tests/test_parser.py

```diff
@@ -853,6 +853,20 @@ def test_fit(self):
         self.checkValue(parser, "/data[50]/field1", 111)
         self.checkValue(parser, "/data[50]/field2", 96)
 
+    def test_arj(self):
```

Comment:
It looks like ARJ is an older archive format used primarily by DOS. Do you have any more "archival" samples, e.g. compressed using the original ARJ program on DOS? These examples look like they were generated on a UNIX-like system.